curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-k.md

```diff
@@ -12,7 +12,7 @@ allowtransparency="true" class="cp_embed_iframe " frameborder="0" height="400" w
 
 To change the placement of items along the cross axis use `align-items`. Try getting the boxes to the center of the container by adding `align-items: center` to `.container`. The desired result looks like this:
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/08.png" alt="an image displaying three blue squares which are centered in the middle of the flex container" style="margin: 15px 0">
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/flex-box/flexbox-06.png" alt="an image displaying three blue squares which are centered in the middle of the flex container" style="margin: 15px 0">
```

Comment:
This isn't bad. Definitely get rid of "an image displaying". You could probably get rid of "which are" after "three blue squares".